tests/test_readers.py

```diff
@@ -121,12 +121,12 @@ def test_nan_values_ris():
     records = _from_file(fp)
 
     # Check missing titles
-    assert records[1].title is None
-    assert records[3].title is None
+    assert records[1].title == ""
+    assert records[3].title == ""
 
     # Check missing abstracts
-    assert records[0].abstract is None
-    assert records[2].abstract is None
+    assert records[0].abstract == ""
+    assert records[2].abstract == ""
```

Comment:
I don't fully understand where this change comes from (as there are no changes to the title and abstract in the record file, isn't it?). I think it has many consequences on the front end. 

In my opinion, it should be `None` when there is no value. This makes many downstream tasks more intuitive, easier, and maintainable.

Reply:
In `record.py:L120` I added a validator on the record that tests if certain fields are a string, and this validator also transforms `None` to `""`. That's where the change comes from.

My thought was that it would be easier in the backend to always be certain that title and abstract are strings. For example, when creating the feature matrix I can just concatenate the values for title and abstract, I don't need to fill the missing values with an empty string.

But I'm fine with changing missing values to `None`. The only thing I would like to do then is replace `""` by `None`, so that you don't get a mix of `None` and `""` for missing values.

Reply:
Thanks, let's go for `None` then.